### PR TITLE
Adds the in-flight Chef Workstation App doc page

### PR DESCRIPTION
**Pull request by marcparadise**

### CODE REVIEW

File: www/site/content/docs/chef-workstation/chef-workstation-app.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+enable=false
+```
+
+## Setting update channel
+
+To set the update channel used for update checking, bring up the CWA tray app menu (Windows/Linux) or the application menu (Mac) and select "About Chef Workstation".
+
+Select the "Channel" button and choose your preferred channel.  This will trigger an immediate update check.
+
+TODO: screen captures of controlling in-app
```
Comment (susanev) [inline www/site/content/docs/chef-workstation/chef-workstation-app.md]: Do we have a need for screenshots here? I think the text is pretty straightforward .. thoughts?

Comment (marcparadise) [reply www/site/content/docs/chef-workstation/chef-workstation-app.md]: I'm alright without them.  It is pretty straightforward.

Comment (kagarmoe) [reply www/site/content/docs/chef-workstation/chef-workstation-app.md]: Is the icon the same on all systems? If so, we could use that. It is nice to know what to look for.

Comment (susanev) [reply www/site/content/docs/chef-workstation/chef-workstation-app.md]: yes, the icon is the same.

File: www/site/content/docs/chef-workstation/chef-workstation-app.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+* Update checking and notifications
+* Chef Workstation version information
+
+Additional features and integrations will be rolled out in future updates.
+
+## Running the Chef Workstation App
+
+### Windows
+
+Start Chef Workstation App from "Recently Added" shortcuts in the Start menu, or the Chef Workstation folder.
```
Comment (jonsmorrow) [inline www/site/content/docs/chef-workstation/chef-workstation-app.md]: shortcut?

File: www/site/content/docs/chef-workstation/chef-workstation-app.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+To set the update channel used for update checking, bring up the CWA tray app menu (Windows/Linux) or the application menu (Mac) and select "About Chef Workstation".
+
+Select the "Channel" button and choose your preferred channel. This triggers an immediate update check.
+
+TODO: screen captures of controlling in-app
+
+```toml
+[updates]
+channel="Current"
```
Comment (jonsmorrow) [inline www/site/content/docs/chef-workstation/chef-workstation-app.md]: current should be lowercase

File: www/site/content/docs/chef-workstation/chef-workstation-app.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+To set the channel used for update checking, bring up the CWA tray app
+menu (Windows/Linux) or the application menu (Mac) and select "About Chef
+Workstation".
+
+Select the "Channel" button and choose your preferred channel.  This will trigger an immediate update check.
+
+
+```
+[updates]
+channel="Current"
```
Comment (kagarmoe) [inline www/site/content/docs/chef-workstation/chef-workstation-app.md]:
```suggestion
channel="current"
```

File: www/site/content/docs/chef-workstation/chef-workstation-app.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+* Update checking and notifications
+* Chef Workstation version information
+
+Additional features and integrations will be rolled out in future updates.
+
+## Running the Chef Workstation App
+
+### Windows
+
+Start Chef Workstation App from "Recently Added" shortcuts in the Start menu,
```
Comment (kagarmoe) [inline www/site/content/docs/chef-workstation/chef-workstation-app.md]:
```suggestion
Start Chef Workstation App from the "Recently Added" shortcut in the Start menu,
```

File: www/site/content/docs/chef-workstation/chef-workstation-app.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+post-install message shown when you install the Chef Workstation package from
+the terminal will tell which -- if any -- additional libraries are required to
+run Chef Workstation App.
+
+### Mac
+Start `Chef Workstation App` from your Applications folder.
+
+## Disabling Automatic Update Checks
+
+To disable automatic update checking of  the update channel of CWA, add or
```
Comment (kagarmoe) [inline www/site/content/docs/chef-workstation/chef-workstation-app.md]:
```suggestion
To disable CWA's automatic updates, add or
```

File: www/site/content/docs/chef-workstation/chef-workstation-app.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+modify the `enable` setting under `updates` in [config.toml](TODO: link):
+
+```
+[updates]
+enable=false
+```
+
+## Setting update channel
```
Comment (kagarmoe) [inline www/site/content/docs/chef-workstation/chef-workstation-app.md]:
```suggestion
## Setting the Update Channel
```

**Pull request merged**